oaknational/Oak-Web-Application
# Feat/227 access beta

## Description

- Adds link to beta in navbar
- Adds new dismissible card variant
- Puts that with link to beta on homepage
- Vanishes when closed
- Webinar placeholder text

## Issue(s)

Fixes #227 

## How to test

1. Go to https://samara-git-feat-227-access-beta-oak-national-academy.vercel.app/
2. Click on link in navbar
3. You should see beta onboarding page
4. Go back
5. Click on "try beta" card
6. You should see beta onboarding page
7. Go back
8. Press X button on beta card - card vanishes
9. Reload - card reappears

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
